app/controllers/placements_controller.rb

```diff
@@ -45,6 +45,10 @@ def index
     # @cart_placement = current_cart.cart_placements.new
     @cart = Cart.new
   end
+  
+  def all
+    @placements_all = Placement.all
```

Comment:
I'm quite confused as to why this isn't called the `index` method and why it's using `@placements_all` instead of `@placements`. 

I think it makes more sense to either include both the search and the list of all placements on one view or to use the `index` in the normal way and to then add a collection for `search`.